### PR TITLE
refactor: command cli tools

### PR DESCRIPTION
**Pull request by blust**
update: Optimize CMD command line

### CODE REVIEW

File: apioak/cmd/quit.lua [hunk 1]
```diff
@@ -0,0 +1,14 @@
+local nginx_signals = require "apioak/cmd/utils/nginx_signals"
+
+local lapp = [[
+Usage: apioak quit
+]]
+
+local function execute(args)
```
Comment (zanehy) [inline apioak/cmd/quit.lua]: Why add a parameter？

File: apioak/cmd/env.lua [hunk 2]
```diff
@@ -0,0 +1,140 @@
...
+        table.insert(db_tables, res[i][table_field])
+    end
+    if table.sort(db_tables) == table.sort(conf_tables) then
+        print("Database Tables        ...OK")
+    else
+        print("Database Tables        ...FAIL")
+    end
+end
+
+local function execute(args)
```
Comment (zanehy) [inline apioak/cmd/env.lua]: Why add a parameter？

File: apioak/cmd/reload.lua [hunk 3]
```diff
@@ -0,0 +1,14 @@
+local nginx_signals = require("apioak/cmd/utils/nginx_signals")
+
+local lapp = [[
+Usage: apioak reload
+]]
+
+local function execute(args)
```
Comment (zanehy) [inline apioak/cmd/reload.lua]: Why add a parameter？

File: apioak/cmd/restart.lua [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+
+local lapp = [[
+Usage: apioak restart
+]]
+
+local function execute(args)
+
+    pcall(stop.execute, args, {quiet = true})
+
+    pcall(start.execute)
```
Comment (zanehy) [inline apioak/cmd/restart.lua]: Why not call nginx_signals directly like other actions?

File: apioak/cmd/stop.lua [hunk 5]
```diff
@@ -0,0 +1,14 @@
+local nginx_signals = require "apioak/cmd/utils/nginx_signals"
+
+local lapp = [[
+Usage: apioak stop
+]]
+
+local function execute(args)
```
Comment (zanehy) [inline apioak/cmd/stop.lua]: Why add a parameter？

File: apioak/cmd/restart.lua (hunk 4 above)
Comment (blust) [reply apioak/cmd/restart.lua]: It is hoped to reuse the capabilities of "cmd.Stop" and "cmd.Start" to be expanded.

File: apioak/cmd/stop.lua (hunk 5 above)
Comment (blust) [reply apioak/cmd/stop.lua]: I'll revise it right away.

File: apioak/cmd/reload.lua (hunk 3 above)
Comment (blust) [reply apioak/cmd/reload.lua]: I'll revise it right away.

File: apioak/cmd/env.lua (hunk 2 above)
Comment (blust) [reply apioak/cmd/env.lua]: I'll revise it right away.

File: apioak/cmd/quit.lua (hunk 1 above)
Comment (blust) [reply apioak/cmd/quit.lua]: I'll revise it right away.

**Comment by zanehy** [comment]
LGTM

**Pull request merged**